packages/sources/tradingeconomics-test/src/config/index.ts

```diff
@@ -0,0 +1,30 @@
+export const defaultEndpoint = 'price'
+export const customSettings = {
+  API_ENDPOINT: {
+    description: 'The HTTP URL to retrieve data from',
+    type: 'string',
+    default: 'https://api.tradingeconomics.com/markets',
+    required: true,
+  },
+  WS_API_ENDPOINT: {
+    description: 'The WS URL to retrieve data from',
+    type: 'string',
+    default: 'wss://stream.tradingeconomics.com/',
+    required: true,
+  },
+  API_CLIENT_KEY: {
+    description: 'The TradingEconomics API client key',
+    type: 'string',
+    required: true,
+    sensitive: true,
+  },
+  API_CLIENT_SECRET: {
+    description: 'The TradingEconomics API client secret',
+    type: 'string',
+  },
```

Comment:
I think there was a mix up between the two env vars. For `API_CLIENT_KEY`, we only need to mark it as required but `API_CLIENT_SECRET` we need to mark as required and sensitive
```suggestion
  API_CLIENT_KEY: {
    description: 'The TradingEconomics API client key',
    type: 'string',
    required: true,
  },
  API_CLIENT_SECRET: {
    description: 'The TradingEconomics API client secret',
    type: 'string',
    required: true,
    sensitive: true,
  },
```